CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## July 6, 2021 v3.0.0
```

Comment:
Not sure about the versioning scheme/number here, but I figured that this would be a major revision since we're breaking the current functionality?

Reply:
to do a major version release you'll need to also change the module pseudoversion in the `go.mod` file and in any references to the library internally (probably in tests). Right now it's `github.com/customerio/go-customerio/v2` it would need to be `/v3` everywhere

TBH this doesn't feel like a large enough change to be worth a major bump, even though it's a breaking change it is breaking a somewhat niche feature -- but that's largely more driven by my annoyances around module versioning. So LGTM either way

Reply:
@hownowstephen I've made those changes.

Reply:
@glosier I couldn't find anywhere in our docs that reference the import line for `go-customerio` but in case there are, they need to be updated to `/v3` as well

@swastik https://fly.customer.io/env/1/transactional/2/setup/review transactional apparently does in-app, and wasn't updated for v2 even :joy: can we get that fixed?

Reply:
Yes, I'll fix it tomorrow.